pkg/api/v1beta3/conversion.go

```diff
@@ -816,15 +816,36 @@ func init() {
 			if err := s.Convert(&in.Lifecycle, &out.Lifecycle, 0); err != nil {
 				return err
 			}
+			out.TerminationMessagePath = in.TerminationMessagePath
+			out.ImagePullPolicy = newer.PullPolicy(in.ImagePullPolicy)
+			// in will always have a securityContext set by the defaulting functions
+			out.SecurityContext = &newer.SecurityContext{}
 			if err := s.Convert(&in.SecurityContext, &out.SecurityContext, 0); err != nil {
 				return err
 			}
-			out.TerminationMessagePath = in.TerminationMessagePath
-			out.Privileged = in.Privileged
-			out.ImagePullPolicy = newer.PullPolicy(in.ImagePullPolicy)
+
+			return nil
+		},
+		func(in *SecurityContext, out *newer.SecurityContext, s conversion.Scope) error {
+			// this is always set by the defaulting functions, no nil check needed
+			out.Capabilities = &newer.Capabilities{}
 			if err := s.Convert(&in.Capabilities, &out.Capabilities, 0); err != nil {
 				return err
 			}
+			// this is always set by the defaulting functions, no nil check needed
+			priv := *out.Privileged
+			out.Privileged = &priv
+
+			if in.SELinuxOptions != nil {
```

Comment:
not necessary with the idea of if in.SELinuxOptions is nil that out.SELinuxOptions will also be set to nil?  But I still need the next line initializing a holder on the parent object right?  If I didn't do that for container.SecurityContext it would convert and then disappear, it seemed